pcsx2/GS/Renderers/DX11/GSDevice11.cpp

```diff
@@ -2521,43 +2509,43 @@ void GSDevice11::RenderHW(GSHWDrawConfig& config)
 
 	GSVector2i rtsize = (config.rt ? config.rt : config.ds)->GetSize();
 
-	GSTexture* hdr_rt = g_gs_device->GetHDRTexture();
+	GSTexture* colcip_rt = g_gs_device->GetColorClipTexture();
```

Comment:
```suggestion
	GSTexture* colclip_rt = g_gs_device->GetColorClipTexture();
```
(And all the uses of it)